mautic/mautic
# Update builder.css with Mautic Color Scheme

Beautify builder.css for better usability and to see future changes in css with each PR.
Update builder.css with Mautic specific colors for the builder.

#### Description:
The builder.css file was a minified file which didn't make sense - the grapesjs-min.css file gets created with the mautic cache and css is served from there anyway.
For future changes this is easier to compare changes and don't change the whole file (because the minfied file consists of only one line - therefor it is difficult to find the changes in it)



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. create a new email/landingpage and open the builder
3. Play around with the builder functions
4. Everything should be visible and look more Mautic-like



<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10431"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
